### PR TITLE
Adding Leesa to users.yml

### PR DESCRIPTION
**Pull request by jackofseattle**
Thanks for making a very helpful tool - saves tons of time in code reviews!

### CODE REVIEW

File: website/data/users.yml [hunk 1]
```diff
@@ -177,3 +177,6 @@
 - caption: 3YOURMIND
   image: "/images/users/3yourmind.svg"
   infoLink: https://www.3yourmind.com/
+- caption: Leesa Sleep
+  image: "/images/users/leesa.svg"
```
Comment (lipis) [inline website/data/users.yml]: no need for the `"`

Comment (lipis) [reply website/data/users.yml]: maybe you can also remove them from line 178 as well..

Comment (lydell) [reply website/data/users.yml]: Meh, let's just merge and deal with quotes some other time.

**Comment by j-f1** [comment]
Thanks for using Prettier!

**Pull request merged**